src/bin/efreet/efreetd_cache.c

```diff
@@ -518,7 +518,7 @@ desktop_changes_monitor_add(const struct stat *st, const char *path)
    const char *monpath = path;
 
    if (eina_hash_find(desktop_change_monitors, path)) return;
-#ifndef _WIN32
+#if defined(S_ISLNK) && !defined(_MSC_VER)
```

Comment:
This `#ifndef _WIN32` is not ours, so I think it is okay to don't change anything here.